apps/sms/services/js/messaging/messaging_service.js

```diff
@@ -12,7 +13,7 @@ if (!('BridgeServiceMixin' in self)) {
 (function(exports) {
 
 const SERVICE_NAME = 'messaging-service';
-const MOBILE_MESSAGE_CLIENT_NAME = 'mozMobileMessageShim';
+const MOBILE_MESSAGE_CLIENT_NAME = 'moz-mobile-message-shim';
```

Comment:
No matter which style we apply it must be horrible considering the channel and possibly app instance here :) Just want to make sure we have the same naming convention here.